Update installing-addons.md (#781)
ember-learn/guides-source: guides/v3.3.0/tutorial/installing-addons.md

We also specified a `namespace` of `/api` in our mirage configuration.
Without this change, navigation to `/rentals` in our application would conflict with Mirage.

> In order for this to work, we need our application to default making requests to the namespace of `/api`.

Reviewer: ```suggestion
In order for this to work, we need to tell the application to send its requests to the `/api` namespace by default.
```

Author: Aaaaaawesome! ;-)
